debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -1150,6 +1167,96 @@ public static LsnFlushTimeoutAction parse(String value, String defaultValue) {
             .withDefault(false)
             .withValidation(Field::isBoolean);
 
+    public enum OffsetSlotMismatchStrategy implements EnumeratedValue {
+        /**
+         * Do not validate or seek slot/offset positions (legacy behavior).
+         * Attempts to stream from the stored offset without validating against the replication slot.
+         * Slot recreation is not detected. Replaces slot.seek.to.known.offset.on.start = false.
```

Comment:
```suggestion
         * Slot recreation is not detected. Replaces internal.slot.seek.to.known.offset.on.start = false.
```

The config option was an internal one so its name is `internal.slot.seek.to.known.offset.on.start`. This should be reflected in the PR.